server/server.go

```diff
@@ -20,12 +20,26 @@ type cribbageServer struct {
 func (cs *cribbageServer) Serve() {
 	router := gin.Default()
 
+	router.LoadHTMLGlob("templates/*")
+	router.Static("/assets", "./assets")
```

Comment:
Somewhere on the internet is suggested to put all of our assets in an `assets/` directory, then putting sub directories with the file type. that's why we have js and wasm in the folders we do

Reply:
🌵 📖 we use backticks here, right?